src/Compilers/VisualBasic/Test/Semantic/Semantics/UserDefinedBinaryOperators.vb

```diff
@@ -2546,6 +2546,93 @@ End Module
+    <file name="a.vb"><![CDATA[
+Public Class Foo
+    Public Shared Operator Not(Optional x As Foo) As Boolean 'BIND1:"Foo"
+        Return True
+    End Operator
+End Class
+    ]]></file>
+</compilation>
+
+            Dim compilation = CreateCompilationWithMscorlibAndVBRuntime(compilationDef)
+            Dim model = GetSemanticModel(compilation, "a.vb")
+
+            Dim typeSyntax = CompilationUtils.FindBindingText(Of IdentifierNameSyntax)(compilation, "a.vb", 1)
+            Dim typeSymbolInfo = model.GetSymbolInfo(typeSyntax)
+            Assert.NotNull(typeSymbolInfo.Symbol)
+            Dim typeSymbol = TryCast(typeSymbolInfo.Symbol, NamedTypeSymbol)
+            Assert.NotNull(typeSymbol)
+            Assert.Equal("Foo", typeSymbol.Name)
+
+            CompilationUtils.AssertTheseDiagnostics(compilation,
+<expected>
+BC33010: 'operator' parameters cannot be declared 'Optional'.
+    Public Shared Operator Not(Optional x As Foo) As Boolean 'BIND1:"Foo"
+                               ~~~~~~~~
+BC30201: Expression expected.
+    Public Shared Operator Not(Optional x As Foo) As Boolean 'BIND1:"Foo"
+                                                ~
+BC30812: Optional parameters must specify a default value.
+    Public Shared Operator Not(Optional x As Foo) As Boolean 'BIND1:"Foo"
+                                                ~
+</expected>)
+        End Sub
+
+        <WorkItem(14688, "https://github.com/dotnet/roslyn/issues/14688")>
+        <Fact>
+        Public Sub OperatorsWithDefaultParameters_WithValue()
+            Dim compilationDef =
+<compilation name="OperatorsWithDefaultValuesAreNotBound">
+    <file name="a.vb"><![CDATA[
+Public Class Foo
+    Dim Property1 As Foo
+    Public Shared Operator Not(Optional x As Foo = Property1) As Boolean 'BIND1:"Foo" 'BIND2:"= Property1"
+        Return False
+    End Operator
+End Class
+    ]]></file>
+</compilation>
+
+            Dim compilation = CreateCompilationWithMscorlibAndVBRuntime(compilationDef)
+            Dim model = GetSemanticModel(compilation, "a.vb")
+
+            Dim typeSyntax = CompilationUtils.FindBindingText(Of IdentifierNameSyntax)(compilation, "a.vb", 1)
+            Dim typeSymbolInfo = model.GetSymbolInfo(typeSyntax)
+            Assert.NotNull(typeSymbolInfo.Symbol)
+            Dim typeSymbol = TryCast(typeSymbolInfo.Symbol, NamedTypeSymbol)
+            Assert.NotNull(typeSymbol)
+            Assert.Equal("Foo", typeSymbol.Name)
+
+            Dim equalsSyntax = CompilationUtils.FindBindingText(Of EqualsValueSyntax)(compilation, "a.vb", 2)
+            Assert.NotNull(equalsSyntax.EqualsToken)
+            Assert.Equal("=", equalsSyntax.EqualsToken.Value)
+
+            Assert.NotNull(equalsSyntax.Value)
+            Dim valueSyntax = TryCast(equalsSyntax.Value, IdentifierNameSyntax)
+            Assert.NotNull(valueSyntax)
+            Dim valueSymbolInfo = model.GetSymbolInfo(valueSyntax)
+            Assert.NotNull(valueSymbolInfo.Symbol)
```

Comment:
>Assert.NotNull(valueSymbolInfo.Symbol) [](start = 12, length = 38)

It would be sufficient to test ```Assert.Equal("...", valueSymbolInfo.Symbol.ToTestDisplayString())```, it would cover this and all asserts below. This is just an FYI.